pages/events/[event].tsx

```diff
@@ -27,12 +37,12 @@ const EventPage: NextPage = () => {
   const { supabase, user } = useSupabase();
   const [dataFetchErrors, setDataFetchErrors] = useState<string[]>([]);
   const [eventData, setEventData] = useState<Event | null>();
-
-  const [countdown, setCountdown] = useState(10);
+  const [dateStatus, setDateStatus] = useState<DateStatus>(DateStatus.loading);
+  const [presentEventCountdown, presentEventsetCountdown] = useState(10);
```

Comment:
I'd be happy to keep these names as is, I think it's pretty clear that the countdown is only used for the present event